pysumma/simulation.py

```diff
@@ -88,132 +120,105 @@ def run_docker(self, run_suffix=None, processes=1,
         cmd = ''.join(['docker run -v {}:{}'.format(fman_dir, fman_dir),
                        ' -v {}:{}'.format(settings_path, settings_path),
                        ' -v {}:{}'.format(input_path, input_path),
-                       ' -v {}:{}'.format(output_path, output_path),
-                       '/bin/bash -c "',
+                       ' -v {}:{} '.format(output_path, output_path),
+                       " --entrypoint '/bin/bash' ",
+                       self.executable,
```

Comment:
need to define `self.executable` like `self.executable = "uwhydro/summa:sopron_2018"`

Reply:
You should be able to give `"uwhydro/summa:sopron_2018"` as the `xecutable` when constructing the `Simulation` instance. So like this:

```
s = ps.Simulation('uwhydro/summa:sopron_2018', file_manager)
s.run('docker', 'docker_test')
```